homeassistant/components/sma_manager/const.py

```diff
@@ -0,0 +1,9 @@
+"""Component Constants."""
+
+DOMAIN = "sma_manager"
+
+# Config Flow Constants
+CONF_NAME = "name"
+CONF_HOST = "host"
+CONF_PORT = "port"
+CONF_REFRESH_INTERVAL = "refresh_time"
```

Comment:
These already exist in homeassistant/const.py

Reply:
Done